x/ibc-rate-limit/contracts/rate-limiter/src/state.rs

```diff
@@ -119,6 +122,8 @@ impl From<&QuotaMsg> for Quota {
     }
 }
 
+// TODO: Add docs that this is the main tracker, we should be setting multiple of these per contract
+// Q: Should we rename to "ChannelRateLimiter", and rename flow to "FlowTracker"?
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct ChannelFlow {
```

Comment:
What do you think about moving `Channel_flows` then `channelFlow` definition to the top of this file?

Reply:
Usually cosmwasm contracts have the state definition at the bottom of the state file, but we could move this around or maybe split the data structures into a separate file